src/google-maps/README.md

```diff
@@ -0,0 +1,131 @@
+# Angular Google Maps component
+
+This component provides a Google Maps angular component that implements the
+[Google Maps JavaScript API](https://developers.google.com/maps/documentation/javascript/tutorial).
+File any bugs against the [angular/components repo](https://github.com/angular/components/issues).
+
+## Installation
+
+To install, run `npm install @angular/google-maps`.
+
+## Loading the API
+
+- First follow [these steps](https://developers.google.com/maps/gmp-get-started) to get an API key that can be used to load Google Maps.
+- Make sure that the API is loaded before bootstrapping the GoogleMap component. See the [example](#example) below.
+
+## GoogleMap
+
+The GoogleMap component wraps the [google.maps.Map class](https://developers.google.com/maps/documentation/javascript/reference/map) from the Google Maps JavaScript API. Settings for the map are set using inputs. The `options` input accepts a [google.maps.MapOptions object](https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions), and there are convenience inputs for just setting the `center` and `zoom` of the map if you don't want to create an entire google.maps.MapOptions object. There are also `height` and `width` inputs that accept a string to set the size of the Google map. [Events](https://developers.google.com/maps/documentation/javascript/reference/map#Map.bounds_changed) are set using the outputs of the GoogleMap component, although events that return a MouseEvent have been given a "map" prefix so as to not interfere with Angular's native mouse event handling behavior. Other methods and properties set on a google.maps.Map object are available by providing an instance of the GoogleMap component using the [ViewChild decorator](https://angular.io/api/core/ViewChild).
```

Comment:
Add backticks around references to code (e.g. `GoogleMap` and `ViewChild`).

Reply:
Done.